### PR TITLE
enabled static and error checks

### LINKED ISSUE #47: Builds: Static and Error checking of Go code
**Enhancement Description**
In order to facilitate coding best practices, this adapter's code should have static analysis and error checking run as part of its build process.

**Chore Description**
Using the meshery-istio adapter's Actions workflow as an example, enable [Static and Error checks](https://github.com/layer5io/meshery-istio/blob/master/.github/workflows/ci.yml#L26-L51) in this repository's [CI.yml workflow](../blob/master/.github/workflows/ci.yml).

### PR DESCRIPTION
**Pull request by ashiscs**
fixes #47 
Signed-off-by: ashiscs <ashissingh640@gmail.com>

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -22,32 +22,32 @@ jobs:
...
-#     - name: Check out code
-#       uses: actions/checkout@master
-#       with:
-#         fetch-depth: 1
-#     - name: Setup Go
-#       uses: actions/setup-go@v1
-#       with:
-#         go-version: ${{ secrets.GO_VERSION }}
-#     - run: GOPROXY=direct GOSUMDB=off GO111MODULE=on go get -u honnef.co/go/tools/cmd/staticcheck; /home/runner/go/bin/staticcheck -tags draft -checks all ./... # https://staticcheck.io/docs/checks
+   error_check:
```
Comment (leecalcote) [inline .github/workflows/ci.yml]: @girishranganathan what's different here? What needs to be updated in order to enable the new static and error checks for all other adapters?

Comment (leecalcote) [reply .github/workflows/ci.yml]: @girishranganathan bump

Comment (girishranganathan) [reply .github/workflows/ci.yml]: When we started, we had these commented out. This change is just to bring back 2 of those checks in this adapter. Eventually, we will be bringing this change to others as well.

Comment (leecalcote) [reply .github/workflows/ci.yml]: @girishranganathan great. please merge, if appropriate

**Pull request merged**